other/guess_the_number_search.py

```diff
@@ -1,51 +1,51 @@
 """
-Get the random number guessed by the computer by passing the lower, higher, and the
```

Comment:
88 characters per line max and sentences please.

Reply:
Sir, please check the new pushed commit, I've changed the sentences and documentation and  set to the maximum 88 characters